packages/js/client/src/__tests__/Web3ApiClient.spec.ts

```diff
@@ -1818,7 +1818,7 @@ describe("Web3ApiClient", () => {
     );
   });
 
-  it("environment types", async () => {
+  it.only("environment types", async () => {
```

Comment:
What's the reason for using `only` here?

Reply:
Was for testing